snap/snapcraft.yaml

```diff
@@ -36,4 +36,8 @@ parts:
 
       mkdir -p $TARGET
       cp -vr res $TARGET/res
-     
\ No newline at end of file
+     
+      # Change the startup message, for the sake of testing
+      $SNAPCRAFT_PROJECT_DIR/amend-startup-msg.sh $TARGET
```

Comment:
I think adding the custom startup message in the file is better because we wouldn't pollute the example snapcraft for the sake of testing.